pkg/utils/common/common.go

```diff
@@ -66,9 +66,10 @@ const (
 	defaultRetryInterval = 3 * time.Second
 
 	// https://cert-manager.io/docs/trust/trust-manager/#preparing-for-production
-	DefaultOperatorCertSecretName = "activemq-artemis-manager-cert"
-	DefaultOperatorCASecretName   = "activemq-artemis-manager-ca"
-	DefaultOperandCertSecretName  = "broker-cert" // or can be prefixed with `cr.Name-`
+	DefaultOperatorCertSecretName   = "activemq-artemis-manager-cert"
+	DefaultOperatorCASecretName     = "activemq-artemis-manager-ca"
+	DefaultOperandCertSecretName    = "broker-cert" // or can be prefixed with `cr.Name-`
```

Comment:
I think it would be good to also look for a secret with the cr.Name prepended, in the same way as the broker-cert. this will allow isolated config in addition to a shared secret.